src/types/props.ts

```diff
@@ -25,18 +24,18 @@ export default interface IProps {
   readonly queryDelay?: number;
   readonly googleMaps?: any;
   readonly highlightMatch?: boolean;
-  readonly onSuggestSelect?: (suggest?: ILocation) => void;
```

Comment:
I’m unsure about this change — the suggest provided on select may only consist of the location and the optional Google Maps result.